### ISSUE TITLE
Resume yolov8 training if it’s interrupted

### ISSUE THREAD

**Issue by alinaivanovaoff** [keras-team/keras-cv#2511]
Hi everyone,

First of all, thank you for your excellent work on implementing YOLOv8!

I have a question: does your YOLOv8 implementation include an option to resume training if it gets interrupted, similar to the functionality provided by Ultralytics?

> How do I resume training if it’s interrupted?
> YOLOv8 makes it easy to resume training from where it was interrupted by simply using the resume=True flag in your training command.

Looking forward to your response!

**Comment by paraynaud** [comment]
Hello, you can use [callbacks](https://keras.io/api/callbacks/) to save the weights found across your training with the `fit` method.
To do so, take a look at the [ModelCheckpoint](ModelCheckpoint) callback.
The weights saved are monitored by the loss function or the metric you choose.

By using this callback, even if your training is interrupted for whatever reason, you can always start a new training based on the weights you found so far.
There is an option to keep only the best weight you found across the epochs, but you can also store the last weights found.
